tests/manage/pv_services/test_attach_mutliple_pvc_rawblock.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import random
+import pytest
+
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs import constants
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.parametrize(
+    argnames=["reclaim_policy"],
+    argvalues=[
+        pytest.param(
+            constants.RECLAIM_POLICY_DELETE, marks=pytest.mark.polarion_id("OCS-1296")
+        ),
+        pytest.param(
+            constants.RECLAIM_POLICY_RETAIN, marks=pytest.mark.polarion_id("OCS-1296")
+        )
+    ]
+)
+class TestMultiAttachPVC(ManageTest):
+    @pytest.fixture()
+    def namespace(self, project_factory):
+        """
+        Create a project for the test
+
+        """
+        proj_obj = project_factory()
+        self.namespace = proj_obj.namespace
+
+    @pytest.fixture()
+    def storageclass(self, storageclass_factory, reclaim_policy):
+        """
+        Create storage class with reclaim policy
+        """
+        self.reclaim_policy = reclaim_policy
+        self.sc_obj = storageclass_factory(interface=constants.CEPHBLOCKPOOL, reclaim_policy=reclaim_policy)
+
+    def test_multipvc_attach(self, storageclass, namespace, teardown_factory):
+        """
+        Test for attaching multiple pvcs to single pod
+        """
+        worker_nodes = helpers.get_worker_nodes()
+        pvcs = list()
+        for size in ['500Mi', '10Gi', '1Ti']:
+            pvcs.append(helpers.create_pvc(
+                sc_name=self.sc_obj.name, size=size,
+                access_mode=constants.ACCESS_MODE_RWX,
+                namespace=self.namespace,
+                volume_mode='Block'
+            )
+            )
+
+        for pvc in pvcs:
+            helpers.wait_for_resource_state(
+                resource=pvc, state=constants.STATUS_BOUND, timeout=120
+            )
+
+        pvs = [pvc.backed_pv_obj for pvc in pvcs]
```

Comment:
Need reload() for getting pv ?

Reply:
when pvcs are bound, pv would have created right? do we need to reload?

Reply:
backed_pv_obj() is doing that. So not needed to do it here.